From 140e63477013b8ae266366286e3a4809d889e680 Mon Sep 17 00:00:00 2001
From: LiangRongrong <Liangrongrong@mprc.pku.edu.cn>
Date: Tue, 29 Nov 2011 21:40:15 +0800
Subject: [PATCH 097/641] UniCore64:fix checkpatch warning

WARNING: line over 80 characters
extern void flush_tlb_range(struct vm_area_struct *, unsigned long,
unsigned long);

Signed-off-by: Liang Rongrong <liangrongrong@mprc.pku.edu.cn>
---
 arch/unicore64/include/asm/tlbflush.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/unicore64/include/asm/tlbflush.h b/arch/unicore64/include/asm/tlbflush.h
index cb065ad..7d4a474 100644
--- a/arch/unicore64/include/asm/tlbflush.h
+++ b/arch/unicore64/include/asm/tlbflush.h
@@ -8,7 +8,8 @@
 
 extern void flush_tlb_mm(struct mm_struct *);
 extern void flush_tlb_page(struct vm_area_struct *, unsigned long);
-extern void flush_tlb_range(struct vm_area_struct *, unsigned long, unsigned long);
+extern void flush_tlb_range(struct vm_area_struct *, unsigned long,	\
+		unsigned long);
 
 extern void flush_tlb_kernel_range(unsigned long, unsigned long);
 
-- 
1.7.9.5

